app/src/main/java/com/yacgroup/yacguide/WhatsNewInfo.kt

```diff
@@ -63,15 +64,13 @@ class WhatsNewInfo(private var _activity: AppCompatActivity) {
     @SuppressLint("InflateParams")
     fun showDialog() {
         _activity.let {
-            val inflater = it.layoutInflater
-            // Pass null as the parent view because its going in the dialog layout.
-            _view = inflater.inflate(R.layout.whats_new, null)
```

Comment:
Can you remove `_view` as a property now?

Reply:
Good catch.